mobile/mysterium/wireguard_connection_setup.go

```diff
@@ -19,22 +19,158 @@ package mysterium
 
 import (
 	"errors"
+	"os"
+	"sync"
+	"time"
 
+	"github.com/mysteriumnetwork/node/services/wireguard/wgdefinition"
```

Comment:
I have no idea why goimports behaves like that :(

Reply:
It allows for grouping of imports, so if you do click enter anywhere on the imports - it will assume this will be a separate block. It formats the blocks independently :)